src/main/java/com/cloudbees/hudson/plugins/folder/computed/ComputedFolder.java

```diff
@@ -108,6 +108,12 @@ protected ComputedFolder(ItemGroup parent, String name) {
         init();
     }
 
+    protected ComputedFolder(ItemGroup parent, String name, OrphanedItemStrategy strategy) {
```

Comment:
@jglick To use `setOrphanedItemStrategy` was my first approach this weekend but while a discussed in private with @amuniz, I decided to change it, especially, for `/*almost final*/`.